db/seeds.rb

```diff
@@ -5,3 +5,9 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+skills = Skill.create([{ name: 'skill1', featured: true},
+                       { name: 'skill2', featured: true},
+                       { name: 'skill3', featured: false},
+                       { name: 'skill4', featured: true},
+                       { name: 'skill5', featured: false},
```

Comment:
Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.